cmd/openfga/openfga_test.go

```diff
@@ -43,14 +43,7 @@ func TestBuildServerWithPresharedKeyAuthenticationFailsIfZeroKeys(t *testing.T)
 	os.Setenv("OPENFGA_AUTH_PRESHARED_KEYS", "")
 
 	_, err := buildService(noopLogger)
-	if err == nil {
-		t.Fatal("Expected to fail with error")
-	}
-
-	expectedError := "invalid auth configuration, please specify at least one key"
-	if !strings.Contains(err.Error(), expectedError) {
-		t.Fatalf("Expected to fail with error %v but got %v", expectedError, err.Error())
-	}
+	require.EqualError(t, err, "failed to initialize authenticator: invalid auth configuration, please specify at least one key")
```

Comment:
Do we want to do error message comparison, or just simply `require.Error(t, err)`?

I'm not a huge fan of checking error messages as they can change and aren't technically part of a stable API surface. I'm fine if we do this, but I think `require.Error(t, err)` is just as good and less flakey (potentially).

Reply:
I like less flakey-ness, but we don't want to pass on catching the wrong error though either. Is there some middle ground?